openlibrary/plugins/openlibrary/js/edit.js

```diff
@@ -235,15 +235,28 @@ export function validateIdentifiers(data) {
 export function initClassificationValidation() {
     initJqueryRepeat();
     const dataConfig = JSON.parse(document.querySelector('#classifications').dataset.config);
+
+    // Prevent form submission on Enter for classification fields
+    $('#select-classification, #classification-value').on('keydown', function(e) {
+        if (e.key === 'Enter') {
+            e.preventDefault();
+            $('#classifications').repeat('add');
```

Comment:
This appears not to be working for me; this is I think the line we want:

```suggestion
            $('#classifications .repeat-add').click();
```